wbooks/src/app/utils/Constants.js

```diff
@@ -1,2 +1,5 @@
 export const DEFAULT_AUTH = '/';
 export const DEFAULT_NO_AUTH = '/login';
+const accessToken = localStorage.getItem('access_token')
+
+export default accessToken;
```

Comment:
Use a named export, not the default.
Also, I would make this a function in a service.
Making it a const could be problematic.